doc/tutorials/content/sources/ensenso_cameras/ensenso_cloud_images_viewer.cpp

```diff
@@ -26,6 +26,7 @@ typedef pcl::PointCloud<PointT> PointCloudT;
 
 /** @brief Convenience typdef for the Ensenso grabber callback */
 typedef std::pair<pcl::PCLImage, pcl::PCLImage> PairOfImages;
+typedef boost::shared_ptr<PairOfImages> PairOfImagesPtr;
```

Comment:
You've defined the extra `PairOfImagesPtr` here although all subsequent changes then use `PairOfImages::Ptr`. Was this definition necessary?